tests/test_fastapi_users.py

```diff
@@ -22,12 +22,21 @@ async def test_app_client(
         UserDB,
     )
 
+    def verification_callback():
+        pass
+
     app = FastAPI()
     app.include_router(fastapi_users.get_register_router())
     app.include_router(fastapi_users.get_reset_password_router("SECRET"))
     app.include_router(fastapi_users.get_auth_router(mock_authentication))
     app.include_router(fastapi_users.get_oauth_router(oauth_client, "SECRET"))
     app.include_router(fastapi_users.get_users_router(), prefix="/users")
+    app.include_router(
+        fastapi_users.get_verify_router(
+            after_verification_request=verification_callback,
+            verification_token_secret="teststring",
```

Comment:
Let's use `"SECRET"` to be consistent with the others.

Reply:
Yep sure!